### PR TITLE
render oauth url

### PR DESCRIPTION
**Pull request by JarbasAl**
related to https://github.com/OpenVoiceOS/ovos-PHAL-plugin-oauth/issues/16

### CODE REVIEW

**Comment by JarbasAl** [comment]
any other ideas how we can improve oauth in a mk1? theres several stages where we could render some icon in the faceplate

ideas welcome, an editor can be found here http://htmlpreview.github.io/?https://raw.githubusercontent.com/MycroftAI/enclosure-mark1/master/editor.html

**Comment by builderjer** [comment]
Just found this too. Can use it with a ssh login. 

https://github.com/vincascm/qrencode

File: ovos_PHAL_plugin_mk1/__init__.py [hunk 1]
```diff
@@ -173,6 +174,10 @@ def handle_register_factory_reset_handler(self, message):
         self.bus.emit(message.reply("system.factory.reset.register",
                                     {"skill_id": "ovos-phal-plugin-mk1"}))
 
+    def handle_oauth_url(self, message):
+        message.data["text"] = message.data["url"]
```
Comment (mikejgray) [inline ovos_PHAL_plugin_mk1/__init__.py:178]: Consider `message.data.get()` or a validation both keys will exist before indexing

File: ovos_PHAL_plugin_mk1/__init__.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def __init__(self, bus=None, config=None):
         self.bus.on("ovos.common_play.stop", self.on_display_reset)
         self.bus.on("mycroft.audio.service.play", self.on_music)
         self.bus.on("mycroft.audio.service.stop", self.on_display_reset)
+        self.bus.on("oauth.url", self.handle_oauth_url)
```
Comment (mikejgray) [inline ovos_PHAL_plugin_mk1/__init__.py:86]: Reminder to update the message spec when this gets merged

**Pull request closed without merging**